aspnetcore/fundamentals/host/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Host in ASP.NET Core
+author: guardrex
+description: Learn about the ASP.NET Core Web Host and .NET Generic Host, which are responsible for app startup and lifetime management.
+manager: wpickett
+ms.author: riande
+ms.date: 05/14/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: fundamentals/host/index
+---
+# Host in ASP.NET Core
+
+.NET apps configure and launch a *host*. The host is responsible for app startup and lifetime management. Two host APIs are available for use:
+
+* [Web Host](xref:fundamentals/host/web-host) &ndash; Suitable for hosting web apps.
```

Comment:
Per the style guide, don't use spaces on either side of an en dash.

Reply:
I know, but they don't really cover this usage. They aren't explicit about using it to set off a definition. It will look ... well ... *bizarre* lol ... not to have spaces around it for this purpose. Consider these renderings ...

* [Web Host](#) &ndash; Suitable for hosting web apps.
* [Web Host](#)&ndash;Suitable for hosting web apps.

That 2nd usage isn't going to work. We could use a space or a colon ...

* [Web Host](#) Suitable for hosting web apps.
* [Web Host](#): Suitable for hosting web apps.

Although we only use en dash/`&ndash;` infrequently (perhaps a couple of dozen times), note that it's very frequently used in the .NET Core doc set with spaces.

@Rick-Anderson @rachelappel Thoughts on this? ... We've kind'a been doing this for a while now.